webpack.config.js

```diff
@@ -66,6 +68,11 @@ const webpackConfig = {
 			// NODE_ENV is used inside React to enable/disable features that should
 			// only be used in development
 			'process.env.NODE_ENV': JSON.stringify( NODE_ENV ),
+
+			// Feature flags!
```

Comment:
How about relying on the Dev Tools Card for now to test changes to the checklist status?
https://github.com/Automattic/jetpack/blob/master/_inc/client/components/dev-card/index.jsx